Library/Homebrew/test/os/dependency_collector_spec.rb

```diff
@@ -0,0 +1 @@
+require "test/dependency_collector"
```

Comment:
This file should instead be filled with the contents of `Library/Homebrew/test/os/linux/dependency_collector.rb` (which should be deleted)